tlslite/utils/format_input.py

```diff
@@ -0,0 +1,21 @@
+# Authors:
+#   Hubert Kario
```

Comment:
Wasn´t sure, what name I have to write here, because you are author of that function